### PR TITLE
store contains does not trigger getobserver #813

### PR DESCRIPTION
**Pull request by AbfrmBlr**

### CODE REVIEW

File: impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java [hunk 1]
```diff
@@ -262,19 +262,13 @@ public OnHeapStore(final Configuration<K, V> config, final TimeSource timeSource
     getObserver.begin();
     StoreEventSink<K, V> eventSink = null;
     try {
-      OnHeapValueHolder<V> mapping = map.get(key);
+      OnHeapValueHolder<V> mapping = getQuiet(key);
```
Comment (ljacomet) [inline impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java]: Given this change, `get` and `getInternal` can be merged

Comment (AbfrmBlr) [inline impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java]: But `internalGet` is getting used in iterator with updateaccess as false.

Comment (ljacomet) [inline impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java]: Right, my bad

**Comment by ljacomet** [comment]
Good after rebase

**Comment by AbfrmBlr** [comment]
rebased

**Pull request merged**